### PR TITLE
Update docs with viewer version and add new viewer interaction

### PR DESCRIPTION
**Pull request by lauramurgatroyd**

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -21,14 +21,14 @@
...
-copyright = '2021, UKRI STFC'
+copyright = '2022, UKRI STFC'
 author = 'Edoardo Pasca and Laura Murgatroyd'
 
 version = re.sub('^v', '', os.popen('git describe').read().strip())
 version = version.split('-')[0]
 
 rst_epilog = """
-.. |cil_viewer_version| replace:: 21.0.1
+.. |cil_viewer_version| replace:: 22.4.0
```
Comment (paskino) [inline docs/source/conf.py:31]: I suppose this could be > 22.2.0

**Comment by paskino** [comment]
can you make a PR to https://github.com/TomographicImaging/iDVC/tree/update_doc instead than to master?

File: docs/source/conf.py (hunk 1 above)
Comment (lauramurgatroyd) [inline docs/source/conf.py:31]:
```suggestion
.. |cil_viewer_version| replace:: 22.2.0
```

**Comment by lauramurgatroyd** [comment]
Updated where the PR is going

File: docs/source/conf.py [hunk 2]
```diff
@@ -21,14 +21,14 @@
...
-copyright = '2021, UKRI STFC'
+copyright = '2022, UKRI STFC'
 author = 'Edoardo Pasca and Laura Murgatroyd'
 
 version = re.sub('^v', '', os.popen('git describe').read().strip())
 version = version.split('-')[0]
 
 rst_epilog = """
-.. |cil_viewer_version| replace:: 21.0.1
+.. |cil_viewer_version| replace:: 22.2.0
```
Comment (lauramurgatroyd) [inline docs/source/conf.py:31]:
```suggestion
.. |cil_viewer_version| replace:: >22.2.0
```

**Pull request merged**